stubs/redis/METADATA.toml

```diff
@@ -1 +1,5 @@
 version = "4.4.0"
+requires = ["types-pyOpenSSL"]
```

Comment:
What does the stub uploader say if we add `cryptography` here? It would be nice to add it if possible, since we're doing imports from `cryptography` directly in the stub, and it's good to avoid implicit dependencies where possible.

Reply:
I agree it should be there and not implied from types-pyOpenSSL. I didn't realize I forgot it.